src/test/scala/uk/gov/nationalarchives/tdr/api/routes/ConsignmentRouteSpec.scala

```diff
@@ -180,21 +191,15 @@ class ConsignmentRouteSpec extends TestContainerUtils with Matchers with TestReq
   "getConsignment" should "return all requested fields" in withContainers {
     case container: PostgreSQLContainer =>
       val utils = TestUtils(container.database)
-      val consignmentId = UUID.fromString("b130e097-2edc-4e67-a7e9-5364a09ae9cb")
-      utils.createConsignment(consignmentId, userId, fixedSeriesId, "TEST-TDR-2021-MTB")
-      val parentId = "7b19b272-d4d1-4d77-bf25-511dc6489d12"
-      val fileOneId = "e7ba59c9-5b8b-4029-9f27-2d03957463ad"
-      val fileTwoId = "42910a85-85c3-40c3-888f-32f697bfadb6"
-      val fileThreeId = "9757f402-ee1a-43a2-ae2a-81a9ea9729b9"
+      utils.createConsignment(defaultConsignmentId, userId, fixedSeriesId, "TEST-TDR-2021-MTB")
 
       val extensionMatch = "txt"
       val identificationBasisMatch = "TEST DATA identification"
       val puidMatch = "TEST DATA puid"
-      val bodyCode = "default-transferring-body-code"
 
-      utils.createFile(UUID.fromString(fileOneId), consignmentId, fileName = "fileOneName", parentId = UUID.fromString(parentId).some)
-      utils.createFile(UUID.fromString(fileTwoId), consignmentId, fileName = "fileTwoName", parentId = UUID.fromString(parentId).some)
-      utils.createFile(UUID.fromString(fileThreeId), consignmentId, fileName = "fileThreeName", parentId = UUID.fromString(parentId).some)
+      utils.createFile(UUID.fromString(fileOneId), defaultConsignmentId, fileName = "fileOneName", parentId = UUID.fromString(parentId).some)
```

Comment:
Is it worth extracting `UUID.fromString(parentId).some` into a variable or a method, it's used quite a bit?